course_catalog/tasks_test.py

```diff
@@ -355,7 +355,7 @@ def test_process_bootcamps(mock_get_bootcamps):
     course = Course.objects.get(course_id="Bootcamp3")
     assert course.platform == PlatformType.bootcamps.value
     assert course.title == "MIT Sports Entrepreneurship Bootcamp"
-    assert course.location == "Hoffenheim, Germany"
+    # assert course.location == "Hoffenheim, Germany"
```

Comment:
You can just remove this line altogether since we won't be adding the field back in

Reply:
Removed